### PR TITLE
feat(i18n): add Simplified Chinese

### PR DESCRIPTION
**Pull request by xcfox**
add Simplified Chinese for `@valibot/i18n`

### CODE REVIEW

**Comment by fabian-hiller** [comment]
Thank you very much for this contribution! Do you know another native speaker who could review your translations for me?

**Comment by fabian-hiller** [comment]
Can you explain the difference between Simplified Chinese (zh-CN) and Chinese (zh)?

**Comment by xcfox** [comment]
> Can you explain the difference between Simplified Chinese (zh-CN) and Chinese (zh)?

Modern Chinese is divided into Simplified Chinese and Traditional Chinese. Simplified Chinese and Traditional Chinese are two ways of writing the same Chinese character. Simplified Chinese is popular in Mainland China, Singapore and Malaysia, while Traditional Chinese is popular in Taiwan Province of China and Hong Kong SAR of China.

现代中文分为简体中文和繁体中文。简体和繁体是同一个汉字的两种写法。简体中文在中国大陆、新加坡、马来西亚流行，繁体中文在中国台湾省、中国香港流行。

現代中文分為簡體中文和繁體中文。簡體和繁體是同一個漢字的兩種寫法。簡體中文在中國大陸、新加坡、馬來西亞流行，繁體中文在中國臺灣省、中國香港流行。

File: packages/i18n/src/zh-CN.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    cuid2:          (issue) => `无效的 Cuid2 格式：输入为 ${issue.received}`,
+    custom:         (issue) => `无效的输入：输入为 ${issue.received}`,
+    customAsync:    (issue) => `无效的输入：输入为 ${issue.received}`,
+    decimal:        (issue) => `无效的十进制格式：输入为 ${issue.received}`,
+    email:          (issue) => `无效的邮箱地址：输入为 ${issue.received}`,
+    emoji:          (issue) => `无效的 emoji 格式：输入为 ${issue.received}`,
+    endsWith:       (issue) => `无效的结尾：预期结尾为 ${issue.expected}，而输入为 ${issue.received}`,
+    equal:          (issue) => `无效的值：预期为 ${issue.expected}，而输入为 ${issue.received}`,
+    excludes:       (issue) => `无效的内容：预期为 ${issue.expected}，而输入为 ${issue.received}`,
+    finite:         (issue) => `无效的无限值：输入为 ${issue.received}`,
```
Comment (0712uu) [inline packages/i18n/src/zh-CN.ts:20]: 这里应该是“有限的”

**Review by 0712uu — CHANGES_REQUESTED** [review]
I am a native speaker of Chinese. I have made some review.

**Comment by xcfox** [comment]
The misstake is fixed

**Comment by fabian-hiller** [comment]
I will review it in the next days!

**Comment by fabian-hiller** [comment]
Could one or both of you also review the `zh-TW` translations in #427?

**Comment by Mini-ghost** [comment]
In fact, Traditional Chinese and Simplified Chinese not only have different writing methods, but they also feature distinct idiomatic expressions.

After reading the Simplified Chinese translation, I believe it is great. Additionally, the PR has been reviewed by Simplified Chinese users, and I believe the results are trustworthy.

**Comment by fabian-hiller** [comment]
Thank you for your review!

**Pull request merged**